backend/src/api/services/question/chapter.model.ts

```diff
@@ -0,0 +1,15 @@
+import mongoose, { Document } from 'mongoose';
+import { chapter } from '../../validations/schemaLibrary.validation.js';
```

Comment:
I am just curious if you used this `chapter`.

Reply:
I don't think I did.